test/e2e/kubectl.go

```diff
@@ -375,6 +375,48 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 		})
 	})
 
+	framework.KubeDescribe("kubectl plugins", func() {
```

Comment:
I've wondered about it yesterday but haven't commented, yet. Uhm... why not having adding this type of test to `make test-cmd`, seems more appropriate. Additionally, I remember we wanted to get rid of some of e2e, since they're heavier than test-cmd.